### ISSUE TITLE
questions about mRNA data in pancan studies

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/datahub#288]
@jjgao commented on [Mon Aug 06 2018](https://github.com/cBioPortal/cbioportal/issues/4591)

There are four mRNA profiles per study, e.g. for ACC below.

STABLE_ID | GENETIC_ALTERATION_TYPE | DATATYPE | NAME | DESCRIPTION | SHOW_PROFILE_IN_ANALYSIS_TAB
-- | -- | -- | -- | -- | --
acc_tcga_pan_can_atlas_2018_mrna | MRNA_EXPRESSION | CONTINUOUS | mRNA Expression from Illumina HiSeq_RNASeqV2 | mRNA Expression from Illumina HiSeq:   https://www.synapse.org/#!Synapse:syn4874822.6 | 0
acc_tcga_pan_can_atlas_2018_rna_seq_v2_mrna_median | MRNA_EXPRESSION | CONTINUOUS | mRNA Expression Batch Normalized/Merged from Illumina HiSeq_RNASeqV2   syn4976369 | mRNA Expression from Illumina HiSeq Batch Normalized:   https://www.synapse.org/#!Synapse:syn4976369 | 0
acc_tcga_pan_can_atlas_2018_mrna_U133_Zscores | MRNA_EXPRESSION | Z-SCORE | mRNA Expression z-Scores (U133 microarray only) | mRNA z-Scores  (U133 microarray   only) compared to the expression distribution of each gene tumors that are   diploid for this gene. | 1
acc_tcga_pan_can_atlas_2018_rna_seq_v2_mrna_median_Zscores | MRNA_EXPRESSION | Z-SCORE | mRNA Expression Zscores Batch Normalized/Merged from Illumina   HiSeq_RNASeqV2 syn4976369 | mRNA Expression Zscores from Illumina HiSeq Batch Normalized:   https://www.synapse.org/#!Synapse:syn4976369 | 1

A few questions:

- Why does `acc_tcga_pan_can_atlas_2018_mrna` have no z-score profiles?
- What's the difference between `acc_tcga_pan_can_atlas_2018_mrna` and `acc_tcga_pan_can_atlas_2018_rna_seq_v2_mrna_median` since they are both RNAseq V2? I randomly checked some and the data seems to the same. Should we remove `acc_tcga_pan_can_atlas_2018_mrna`?
- Why isn't there a non-zscore profile for `acc_tcga_pan_can_atlas_2018_mrna_U133_Zscores`?

**Comment by ritikakundra** [comment]
Just to add: these are the files we currently have in the portal:
There were the following 4 files:
- data_expression_merged.txt (uses mRNA from HiSeq PanCan, batch-normalized)
- data_expression_merged_Zscores.txt (uses mRNA from HiSeq PanCan, batch-normalized)
- data_expression.txt (uses mRNA from HiSeq RNASeqV2, non PanCan normalized)
- data_expression_Zscores.txt (uses U133 microarray only)

We need to rename the metafile for the last one since the values were computed using HiSeq data

**Comment by jjgao** [comment]
Discussed with @yichaoS @ritikakundra and Fransisco. `acc_tcga_pan_can_atlas_2018_mrna` was the original rna_seq data file and `acc_tcga_pan_can_atlas_2018_rna_seq_v2_mrna_median` was the batch corrected one.

And also U133 naming is wrong.

**Comment by jjgao** [comment]
- [ ] remove `acc_tcga_pan_can_atlas_2018_mrna` / `data_expression.txt` (@schultzn please comment if you'd like to keep the non-batch-corrected version or both.)
- [ ] rename file `data_expression_merged.txt` to `data_RNA_Seq_v2_expression_median.txt`; change the ID to `rna_seq_v2_mrna`. 
- [ ] rename file `data_expression_merged_Zscores.txt` to `data_RNA_Seq_v2_mRNA_median_Zscores.txt`; change the ID to `rna_seq_v2_mrna_median_Zscores`
- [ ] rename file `data_expression_Zscores.txt` to `data_mRNA_median_Zscores.txt`; change the ID to `mrna_median_Zscores`; change the name to `mRNA Expression z-Scores (microarray)`
- [ ] add the original microarray profile `data_expression_median.txt` (id: `mrna`, name: `mRNA expression (microarray)`)
- [ ] regenerate case lists.

**Comment by jjgao** [comment]
@yichaoS @ritikakundra I created a checklist above. Please feel free to add new items. Maybe @angelicaochoa can help regenerate the files?

**Comment by jjgao** [comment]
It looks like there was no microarray data at all. `data_expression_Zscores.txt` was generated from `data_expression.txt`, we both will be removed. The checklist has been updated.

**Comment by jjgao** [comment]
@yichaoS @ritikakundra: have the files all pancan studies fixed based on the checklist above?

@n1zea144 how long will it take to rebuild the public database. If it'll take a while, we should consider at least removing the U133 profile from the portal since they are wrong.

![image](https://user-images.githubusercontent.com/840895/47725553-252bc800-dc2f-11e8-844a-43c388736668.png)

**Comment by ritikakundra** [comment]
@jjgao All Pancan studies now show correct RNA profiles.
